frontend/src/main/scala/bloop/cli/Proxy.scala

```diff
@@ -0,0 +1,90 @@
+package bloop.cli
+
+import java.net.URL
+import java.net.MalformedURLException
+
+import scala.util.control.NonFatal
+
+import bloop.engine.State
+
+/**
+  * Proxy helper module
+  * Information came from  https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html
+  */
+object Proxy {
+  /**
+    * Extract environments variables from the given state to set the proxy properties of the jvm.
+    * If environement variable are not found in the state, the property corresponding is cleared,
+    * to let user add and remove env var.
+    * */
+  def setProxyJvmSettings(state : State) : Unit = {
+    //no_proxy environment variable seems not be binded to a protocol.
```

Comment:
Remove space between this and `maybeNoProxy` and add empty space before the start of the comment?